### PR TITLE
define_module

### PR DESCRIPTION
**Pull request by hgscott**
This adds the define_module module to sbol_utilities and test_define_module to test.

define_module takes an SBOL document, checks if all top-level objects have the same namespace (and so constitute a module), and then creates a new SBOL document with the module definition included.

To test the file an example sbol document (test_files/module_in.nt) is used and the output is compared to test_files/module_out.nt.

Using SBOL Factory was causing tests on Python 3.7 to fail. I removed Python 3.7 from the supported versions (setup.py line 28) and the versions for testing (.github/workflows/python-app.yml line 21) and the tests passed.

### CODE REVIEW

File: sbol_utilities/package.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import os
-import glob
-import warnings
+# import glob
```
Comment (jakebeal) [inline sbol_utilities/package.py:2]: If something isn't needed, please delete instead of commenting out.

**Review by jakebeal — CHANGES_REQUESTED** [review]

File: .github/workflows/python-app.yml [hunk 2]
```diff
@@ -17,7 +17,8 @@ jobs:
       matrix:
         # Default builds are on Ubuntu
         os: [ubuntu-latest]
-        python-version: ['3.7', '3.8', '3.9', '3.10']
```
[the remark this answers is not in the document]
Comment (hgscott) [reply .github/workflows/python-app.yml:20]: Done in push "remove python 3.7"

**Comment by bbartley** [comment]
The failure to build on Python 3.7 should be resolved now with the deployment of the SBOLFactory 1.0a11 wheel

**Comment by jakebeal** [comment]
Thank you, @bbartley , looks like this is all passing now.
@hgscott : if you can just clean out the cache files and make those few changes, I think this will be good to merge.

File: sbol_utilities/package.py (hunk 1 above)
Comment (hgscott) [reply sbol_utilities/package.py]: Resolved in commit "remove unneeded packages"

File: sbol_utilities/define_module.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
+import argparse
```
[the remark this answers is not in the document]
Comment (hgscott) [reply sbol_utilities/define_module.py]: I wasn't entirely sure what you meant by this- I moved all the code to package.py and changed everything from "module" to "package" per https://github.com/SynBioDex/SEPs/pull/115. The test is now test_package.

**Review by jakebeal — APPROVED** [review]

**Pull request merged**